lib/mods/replace-get-set.js

```diff
@@ -55,28 +90,50 @@ function isGlobalSet(node) {
 function replaceGlobalGetSetCalls(node, imports) {
   let args;
   let type;
-  let propertyAccessNode;
+  let nodeToReplace;
+
 
-  if (isGlobalGet(node)) {
-    propertyAccessNode = node;
+  if (isGlobalPropertyAccess(node)) { // global get
     type = 'get';
-  } else if (isGlobalSet(node)) {
-    propertyAccessNode = node.getLeft();
+    nodeToReplace = node;
+  } else if (isSet(node) &&
+      isGlobalPropertyAccess(node.getLeft())) { // global set
     type = 'set';
+    nodeToReplace = node.getLeft();
     args = node.getRight().getText();
-  } else {
+  } else if (isWindowGlobal(node)) { // non-property access window get-set
+    const parent = node.getParent();
+
+    if (isSet(parent)) { // non-property access window set
+      type = 'set';
+      nodeToReplace = parent;
```

Comment:
Don't replace parents! Needs a bit of restructuring to be more like the global get/global set cases.

In particular I think the structure that would work is:

- if isGlobalPropertyAccess
- else if isSet && isGlobalPropertyAccess(left)
- else if isWindowGlobal
- else if isSet && isWindowGlobal(left)

Ideally this should allow you to go back to having a single replaceNodeAndUpdateImports call.